app/models/assignment.rb

```diff
@@ -756,6 +756,55 @@ def groups_submitted
     groupings.select(&:has_submission?)
   end
 
+  def get_num_assigned(assignment, ta_id = nil)
```

Comment:
Yea I noticed this as well actually when I was changing it, but wasn't sure if I should stick to the signatures. I've made the changes now :).